Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/SettingsRepository.kt

```diff
@@ -115,4 +117,13 @@ object SettingsRepository {
     fun updateManualKeyRetrievalTime(value: Long) {
         manualKeyRetrievalTime.postValue(value)
     }
+
+    /**
+     * Get the current background priority state.
+     */
+    fun refreshBackgroundPriorityEnabled(context: Context) {
+        val powerManager = context.getSystemService(Context.POWER_SERVICE) as PowerManager
```

Comment:
to be consistent this should be moved to our connectivity helper -> maybe connectivity helper should be renamed to os helper or something like that

Reply:
If we move it to ConnectivityHelper and rename it to OSHelper I´d suggest to also merge the CameraPermissionHelper into it.
Otherwise a PowerManagementHelper might be an option...